tests/accessor_basic/accessor_common.h

```diff
@@ -467,7 +464,7 @@ void check_empty_accessor_constructor_post_conditions(
   // All size queries return 0
   res_acc[res_i++] = testing_acc.byte_size() == 0;
   res_acc[res_i++] = testing_acc.size() == 0;
-  res_acc[res_i++] = testing_acc.max_size() == 0;
+  res_acc[res_i++] = testing_acc.max_size() || std::numeric_limits<size_t>::max;
```

Comment:
Could you please clarify the motivation behind this change?

Reply:
AdaptiveCpp gives the max value for this function call. Given https://github.com/KhronosGroup/SYCL-CTS/issues/1125 and https://github.com/KhronosGroup/SYCL-Docs/issues/332 I think it's okay to make this change?

Reply:
Oh I did not notice I completely typoed it. Will do.